.gitignore

```diff
@@ -0,0 +1,35 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+bin/*
+Dockerfile.cross
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Go workspace file
+go.work
```

Comment:
We can also add `go.work.sum` to .gitignore file as it is go workspace generated file. The same is done in openstack-operator [.gitignore file](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/.gitignore#L39).
```
# generated workspace file
go.work
go.work.sum
```

Reply:
sure i can do that